source/ice_api.c

```diff
@@ -0,0 +1,1824 @@
+/* Standard includes. */
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+/* API includes. */
+#include "ice_api.h"
+
+/* STUN API includes. */
+#include "stun_data_types.h"
+#include "stun_serializer.h"
+#include "stun_deserializer.h"
+
+/*-----------------------------------------------------------------------------------------*/
+
+/* Static Functions. */
+static bool Ice_IsSameIpAddress( StunAttributeAddress_t * pAddr1,
+                                 StunAttributeAddress_t * pAddr2,
+                                 bool checkPort );
+
+static bool Ice_FindCandidateFromIP( IceAgent_t * pIceAgent,
```

Comment:
Still in some functions the assigned and un-assigned declarations are mixed and jumbled together. Let's align them in a way that un-assigned ones are together first followed by the assigned ones.

Reply:
updated. all the assigned declarations follow the un-assigned declarations.